script.js

```diff
@@ -0,0 +1,7 @@
+const radius = 5;
+const area = getCircleArea(radius);
+console.log(area); // 78.53981633974483
+
+function getCircleArea(providedNumber) {
+    return Math.PI * providedNumber * providedNumber;
```

Comment:
you could also use [Math.pow()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/pow) here